src/main/java/androidx/text/Html.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.text
+
+import android.os.Build
+import android.text.Html
+import android.text.Html.ImageGetter
+import android.text.Html.TagHandler
+import android.text.Spanned
+
+/**
+ * Returns a new [Spanned] from [CharSequence] HTML.
+ */
+fun CharSequence.parseAsHtml(flags: Int? = 0): Spanned {
```

Comment:
`flags: Int? = 0` because https://developer.android.com/reference/android/text/Html.html#FROM_HTML_MODE_LEGACY.

Reply:
Could you please explain this behavior in the documentation of the method? It might worth it for users to understand what the API will do on different versions of the platform.